build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -260,6 +272,8 @@ fun removeTeamcityTempProperty() {
     }
 }
 
+fun Project.enableExperimentalTestFiltering() = !setOf("kotlin-dsl", "performance", "smoke-test").contains(name)
```

Comment:
Please also exclude `build-scan-performance` because it's executed in GE pipeline, not ours.

Reply:
It seems like this one is failing: https://builds.gradle.org/viewLog.html?buildId=43056037&buildTypeId=Gradle_Master_Check_GradleBuildSmokeTestsJava16